lsf/readjobs.py

```diff
@@ -85,13 +85,11 @@ def readjobs(args, fast=False):
             elif key in ("submit_time", "start_time", "finish_time"):
                 if val[-1] in "ELXA":
                     val = val[:-2]
-                year = strftime("%Y")
-                job[key] = mktime(strptime(year + " " + val,
-                                           "%Y %b %d %H:%M"))
+                job[key] = mktime(strptime(val,
+                                           "%b %d %H:%M:%S %Y"))
                 if key != "finish_time" and job[key] > time():
```

Comment:
You can remove this entire if statement and block. It was a fix for around new year.

Reply:
@elmar-peise Done.